### PR TITLE
Feat/customize makers

### PR DESCRIPTION
**Pull request by chihung93**
I have added a customize makers for Current location and other location

### CODE REVIEW

**Comment by chihung93** [comment]
Hi @ferranpons , thanks for your response, My google account console is expired now, Im trying to find some key to take a screenshot it.

**Comment by chihung93** [comment]
https://user-images.githubusercontent.com/5199109/202866200-73f97421-ccf0-49b9-b95d-67329e4f18a3.mp4

Hi @ferranpons , I record a video for this PR

File: leku/src/main/java/com/adevinta/leku/LocationPickerActivity.kt [hunk 1]
```diff
@@ -86,6 +80,7 @@ import com.adevinta.leku.locale.SearchZoneRect
 import com.adevinta.leku.permissions.PermissionUtils
 import com.adevinta.leku.tracker.TrackEvents
 import com.adevinta.leku.utils.ReactiveLocationProvider
+import com.google.android.gms.maps.model.*
```
Comment (ferranpons) [inline leku/src/main/java/com/adevinta/leku/LocationPickerActivity.kt:83]: Please don't use wildcard for imports

**Review by ferranpons — APPROVED** [review]
Many thanks for your contribution. Please make the changes suggested and we'll merge the PR.

File: leku/src/main/java/com/adevinta/leku/LocationPickerActivity.kt (hunk 1 above)
Comment (chihung93) [reply leku/src/main/java/com/adevinta/leku/LocationPickerActivity.kt]: I repalced this with 
```
import com.google.android.gms.maps.model.BitmapDescriptor
import com.google.android.gms.maps.model.BitmapDescriptorFactory
import com.google.android.gms.maps.model.LatLng
import com.google.android.gms.maps.model.Marker
import com.google.android.gms.maps.model.CameraPosition
import com.google.android.gms.maps.model.MapStyleOptions
import com.google.android.gms.maps.model.MarkerOptions

```

**Comment by ferranpons** [comment]
@chihung93 You need to correct these Ktlint issues to be able to merge:

Captura de pantalla 2023-02-08 a las 10 21 25

**Comment by chihung93** [comment]
> @chihung93 You need to correct these Ktlint issues to be able to merge:
> 
> Captura de pantalla 2023-02-08 a las 10 21 25
Screenshot 2023-02-08 at 23 59 32


Fixed! Thanks.

**Comment by ferranpons** [comment]
The build is still not passing. Please review the lint issues:

Captura de pantalla 2023-02-20 a las 9 42 34

Checkout the build result here: [https://github.com/AdevintaSpain/Leku/actions/runs/4126569199/jobs/7129972752](https://github.com/AdevintaSpain/Leku/actions/runs/4126569199/jobs/7129972752)

**Comment by chihung93** [comment]
I have run  ./gradlew :leku:ktlintMainSourceSetCheck . I believe that its good to go live.


Screenshot 2023-02-24 at 01 48 58

**Comment by ferranpons** [comment]
@chihung93 Many thanks for your contribution!

**Pull request merged**